### DISCUSSION TITLE
Offline parameters

### DISCUSSION THREAD

**Discussion by rafaalbelda** [HASwitchPlate/openHASP#313] [General]
Great work here!!!
I would like to create an information page in the tft like the **Information** page in the WebUI.

There should be parameters that the system knows and can be displayed offline, but I only found `%ip%` and `%ssid%` and `%hostname%`

Are there other variables with internal parameters?

**Comment by fvanroie** [comment]
There is [currently](https://github.com/HASwitchPlate/openHASP/blob/fe7db71e8b3df4bbe164ede8307bf326655c6d21/src/hasp/hasp_attribute_helper.h#L227-L250):  hostname, mac, ssid, ip, model, version.

**Comment by rafaalbelda** [comment]
Thanks
Where is this defined in the code? I would like to add more values but I could not find where

**Comment by fvanroie** [comment]
They are defined here: https://github.com/HASwitchPlate/openHASP/blob/fe7db71e8b3df4bbe164ede8307bf326655c6d21/src/hasp/hasp_attribute_helper.h#L226-L257

If the variable is a static char[] then you can use `lv_label_set_text_static` which only stores the pointer to the static char[].
Otherwise use `lv_label_set_text` for local or temporary text strings and the text value will be buffered by LVGL.
